source/tests/system/nifgen_driver_api_tests.cpp

```diff
@@ -610,6 +611,26 @@ TEST_F(NiFgenDriverApiTest, OutputModeConfiguredToSeq_CreateAdvancedArbSequenceF
   expect_api_success(status);
 }
 
+TEST_F(NiFgenDriverApiTest, OutputModeConfiguredToArb_CreateWaveformI16_CreatesSuccessfully)
+{
+  const char* channel_name = "0";
+  const ViInt16 waveform_data_array[] = {0, 1, SHRT_MIN, SHRT_MAX};
```

Comment:
We don't check what happens with actual data passed in the system tests. So even if `SHRT_MIN` and `SHRT_MAX` passed here isn't interpreted correctly by server, the test will still pass. The values here are probably inconsequential to result of the test.

I am not suggesting you add validation of data (we don't do that in the test), I am saying you don't need to add min/max here in the data and also remove the statement from your PR description that we are validating mix/max works for int16 arrays through this test.

Reply:
Done.